types/store/shapediverStoreStandardContainers.ts

```diff
@@ -0,0 +1,98 @@
+import {
+	AppBuilderContainerNameType,
+	IAppBuilderContainer,
+} from "../shapediver/appbuilder";
+
+/**
+ * Type representing the standard container names in the app builder.
+ * This is used to filter out the viewport anchor 2d and 3d (and potentially other future anchors).
+ */
+export type AppBuilderStandardContainerNameType =
+	| AppBuilderContainerNameType.Left
+	| AppBuilderContainerNameType.Right
+	| AppBuilderContainerNameType.Top
+	| AppBuilderContainerNameType.Bottom;
+
+/**
+ * List of all standard container names in the app builder.
+ * These are used to identify and manage the layout of the app builder interface.
+ */
+export const AppBuilderStandardContainerNames: readonly AppBuilderStandardContainerNameType[] =
+	[
+		AppBuilderContainerNameType.Left,
+		AppBuilderContainerNameType.Right,
+		AppBuilderContainerNameType.Top,
+		AppBuilderContainerNameType.Bottom,
+	];
+
+export interface IShapeDiverStoreStandardContainers {
+	/**
+	 * The default containers for each position.
+	 * These are the containers that are either specified by the user or created as a fallback.
+	 *
+	 * We store the definition of these containers here, to be able to adjust them later if needed.
+	 */
+	defaultContainers: Record<
+		AppBuilderStandardContainerNameType,
+		IAppBuilderContainer | undefined
+	>;
```

Comment:
Suggestion for doc update: 

The default container state for each of the standard container names. 
This container state is defined by the `AppBuilder` output of the controller session, or the fallback logic in case no `AppBuilder` output exists.